docs/proposals/updates-first-pass.md

```diff
@@ -0,0 +1,205 @@
+# Initial APB Update support
+
+[OBS Update Spec](https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md#updating-a-service-instance)
+
+## Introduction
+
+Updates allow APB authors to mutate existing service instances.
+
+> By implementing this endpoint, service broker authors can enable users
+to modify two attributes of an existing service instance: the service plan
+and parameters. By changing the service plan, users can upgrade or downgrade
+their service instance to other plans. By modifying properties, users can change
+configuration options that are specific to a service or plan
+
+Therefore, the cases we're focused on are:
+
+1) Changing the argument used for a given parameter. Ex: change a "name"
+
+2) Moving from one plan to another.  Ex: Move from non-persistent plan to
+a persisted plan. Another example of this could be moving from a multi-tenant
+environment to a dedicated environment for a service's database.
+
+NOTE: The Service Catalog currently does **not** implement update, so triggering
+these from the UI is not possible. Updates should be considered very "alpha".
+To demonstrate , plan is to enhance `asbcli` for update support.
```

Comment:
@jmrodri @rthallisey @shawn-hurley RE: `asbcli`

It's the butt of our jokes because it was a load of crap python I hammered out on a Friday afternoon for a Monday demo expecting it to die on Tuesday, but it's proven to be a very useful tool for broker authors, and I'm finding myself now wanting to implement update support. @rthallisey implemented bind support because he needed it. It's clear ASB authors want this kind of tool, we should work together to define what we're looking for out of it and make that happen. IMO, python remains perfect for this. I'm going to log an issue so we have a distinct place to air desires for what we want this tool to be, and let's support it given the collective interest.